packages/theme/css/characteristics/text.css

```diff
@@ -59,8 +55,13 @@
   --uitk-text-background-selected: var(--uitk-palette-neutral-highlight);
 
   --uitk-text-primary-foreground: var(--uitk-palette-neutral-primary-foreground);
+  --uitk-text-primary-foreground-hover: var(--uitk-palette-neutral-primary-foreground);
+  --uitk-text-primary-foreground-selected: var(--uitk-palette-neutral-primary-foreground);
   --uitk-text-primary-foreground-disabled: var(--uitk-palette-neutral-primary-foreground-disabled);
+
   --uitk-text-secondary-foreground: var(--uitk-palette-neutral-secondary-foreground);
+  --uitk-text-secondary-foreground-hover: var(--uitk-palette-neutral-secondary-foreground);
+  --uitk-text-secondary-foreground-selected: var(--uitk-palette-neutral-secondary-foreground);
```

Comment:
These aren't mentioned in the changeset

Reply:
Also do they need to be added to the doc page

Reply:
thanks, these also were deleted previously got added back in merge, shouldnt be here!